main.container.registry.tf

```diff
@@ -12,6 +12,7 @@ module "container_registry" {
   private_dns_zone_id                     = local.container_registry_dns_zone_id
   subnet_id                               = local.container_registry_private_endpoint_subnet_id
   tags                                    = var.tags
+  use_zone_redundancy                     = var.use_zone_redundancy ? true : false
```

Comment:
The ternary is redundant since the var is a boolean.

Reply:
updated